### DISCUSSION TITLE
Would it be possible to add something like choosing a random value from an enum?

### DISCUSSION THREAD

**Discussion by Gisbert12843** [effolkronium/random#37] [Ideas]
I'm still fairly new to C++, but it would be a cool addition if this would be implemented somehow as long as its possible ofc..

**Comment by ilqvya** [comment]
Hey, unfortunately C++ does not support reflection so it's not possible to iterate over enum values, since they're a simple integers under the hood

You still can make some crutch like creating an constexpr initializer_list of all enum values and using Random::get(enums_all_values) on it

**Reply by ilqvya** [reply]
https://stackoverflow.com/a/67659256
